src/shared/ListRepo/ReposTable/ReposTable.jsx

```diff
@@ -134,6 +136,9 @@ function ReposTable({ searchValue, owner, sortItem, filterValues = [] }) {
   const { data: ownerData } = useOwner({
     username: owner || userData?.user?.username,
   })
+  const { provider } = useParams()
+  const { data: tierName } = useTier({ provider, owner })
+  const isPublic = tierName === TierNames.TEAM ? true : null
```

Comment:
I would rename this variable a little bit, it is weird to equate if something is public to a tiername 😓. Maybe like `shouldDisplayPublicReposOnly` or something like that?